package.json

```diff
@@ -2,13 +2,13 @@
   "name": "helloworld",
   "version": "1.0.0",
   "description": "Basic Node.js exercises",
-  "main": "../src/index.js",
+  "main": "index.js",
   "scripts": {
-    "start": "node index.js",
+    "start": "@powershell $ENV:DEBUG='mylib:messages' && node ./src/index.js",
```

Comment:
It's better to do not use platform-depending commands in scripts. Take a look at the [cross-env](https://npmjs.com/package/cross-env) module.